operator.index.json

```diff
@@ -75,5 +75,7 @@
   "v2.12.5",
   "v2.12.6",
   "v2.12.7",
-  "v2.12.9"
+  "v2.12.9",
```

Comment:
2.12.9 shouldn't be here

Reply:
it needs removing then, is there a problem with including it even if it is never used?

Reply:
```suggestion
```